caffe2_builders/v140/auto.bat

```diff
@@ -0,0 +1,102 @@
+rem Remove to original folder after script is finished
+set ORIGINAL_DIR=%cd%
+
+rem Build folders are 
+rem %CAFFE2_ROOT%\build\Debug for Debug and
+rem %CAFFE2_ROOT%\build\Release for Release
+set CAFFE2_ROOT=%~dp0%\pytorch
+
+rem Cloning repository if it doesn't exist
+if not exist pytorch (
+    set PYTORCH_BRANCH=windows-shared-build
+    set PYTORCH_REPO=ArutyunovG
+    call  %~dp0%..\..\internal\clone.bat
+    cd %ORIGINAL_DIR%
+)
+
+rem Should build folder be deleted and build start from scratch?
+if NOT DEFINED REBUILD (
+    set REBUILD=0
+)
+
+rem Debug build enabled by default
+if NOT DEFINED BUILD_DEBUG (
+    set BUILD_DEBUG=1
+)
+
+rem Release build enabled by default
+if NOT DEFINED BUILD_RELEASE (
+    set BUILD_RELEASE=1
+)
+
+rem msbuild /m: option value
+if NOT DEFINED MAX_JOBS (
+    set MAX_JOBS=6
+)
+
+if %REBUILD% EQU 1 (
+    if exist %CAFFE2_ROOT%\build (
+        rmdir %CAFFE2_ROOT%\build /s /q
```

Comment:
Consider using `python setup.py clean`?

Reply:
There is no setup.py here, everything is installed as CMake install rules specify.
Explain please, what do you want setup.py to do?

Reply:
%CAFFE2_ROOT% is the path of the pytorch repo, then there should be setup.py under that directory. Run `python setup.py clean` will clean everything that is not tracked in the repo.